app/screens/channel/channel_post_list/channel_post_list.tsx

```diff
@@ -22,6 +22,7 @@ type Props = {
     contentContainerStyle?: StyleProp<ViewStyle>;
     currentTimezone: string | null;
     currentUsername: string;
+    isCRTEnabled: boolean;
```

Comment:
Where is this prop being assigned ?

Reply:
We are passing it to the `PostList` component as it has `index.tsx` with no observables.

Reply:
I get that, but I don't see this being queried in the channel_post_list/index file nor in the channel/index file

Reply:
Anurag this file is screesn/channel/channel_post_list/channel_post_list.tsx yes? This is passing the prop to the component/post_list and that is fine!!!

But where is the channel_post_list component receeiving the prop assigned? Is it magic?

Reply:
screen/channel/channel_post_list/index.ts

Reply:
Not in this PR?

Reply:
Had to go and look for the code, is not part of this PR, was included before and never used or something.

Reply:
Yes, while I was making PRs from the base PR, it got mixed and included with the Websockets + Queries PR. I got confused too because I was looking in the code and thought your question was different and later realised that code was already part of the Gekidou.